feature/experimental/gribi/otg_tests/programming_with_reload/metadata.textproto

```diff
@@ -0,0 +1,16 @@
+# proto-file: github.com/openconfig/featureprofiles/proto/metadata.proto
+# proto-message: Metadata
+
+uuid:  "4f04fb5b-5bc0-4214-a74d-2a30c0433079"
+plan_id:  "TE-11.4"
+description:  "Gribi Programming with reload"
+testbed:  TESTBED_DUT_ATE_4LINKS
+platform_exceptions: {
+  platform: {
+    vendor: ARISTA
+  }
+  deviations: {
+    static_protocol_name: "STATIC"
+    default_network_instance: "default"
+  }
+}
```

Comment:
Please add below deviation for NOKIA 
'''
platform_exceptions: {
   platform: {
    vendor: NOKIA
  }
  deviations: {
    explicit_port_speed: true
    explicit_interface_in_default_vrf: true
    static_protocol_name: "static"
    interface_enabled: true
    }
  }
'''

Reply:
ack